src/web/mjs/connectors/CrunchyManga.mjs

```diff
@@ -1,12 +1,14 @@
 import Crunchyroll from './templates/Crunchyroll.mjs';
+import Manga from '../engine/Manga.mjs';
 
 export default class CrunchyManga extends Crunchyroll {
 
     constructor() {
         super();
+        this.url = "https://www.crunchyroll.com";
```

Comment:
thsi is already set in Crunchyroll.mjs that this extends from so shouldn't be necessary